Program.cs

```diff
@@ -16,7 +16,7 @@ namespace HttpBomb
 {
     class Program
     {
-        const int ONE_SEC_MS = 1000;
+        const int ONE_SEC_MS = 10000000; //Bad code
```

Comment:
This isn't what was requested
```suggestion
        const int ONE_SEC_MS = 10000000; //Bad coding
```